frontend/class-frontend.php

```diff
@@ -1894,12 +1900,9 @@ public function json_ld() {
 	 *
 	 * @since 1.8
 	 *
-	 * @param string $output  The array to be encoded and output
 	 * @param string $context The context of the output, useful for filtering
 	 */
-	protected function json_ld_output( $output, $context ) {
-		echo "<script type='application/ld+json'>";
-
+	protected function json_ld_output( $context ) {
```

Comment:
Verb as first to make it clear it's echoing? `function output_json_ld` is less likely to be confused with method returning something.